Source/Core/EarthOrientationParameters.js

```diff
@@ -81,9 +81,9 @@ import TimeStandard from './TimeStandard.js';
 
             // Download EOP data.
             var that = this;
-            this._downloadPromise = when(resource.fetchJson(), function(eopData) {
+            this._downloadPromise = resource.fetchJson().then(function(eopData) {
```

Comment:
This was very old, no longer correct use of promises that I just switched to the standard approach.